source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -356,6 +356,21 @@ function New-OctopusArtifact
     }
 }
 
+function Update-Progress
+{
+	[CmdletBinding()]
+	param(
+        [int] $percentage, 
+        [Parameter(ValueFromPipeline=$true)][string]$message
```

Comment:
I wouldn't have picked the message as the `ValueFromPipeline`, but I also can't think of a reason not to pick the message.

Reply:
Do yo think we should swap the parameters around? `Update-Process "Hello World" 50`

Reply:
No, I like having the percentage first. :+1: I think it's good for go.